docs/cli.md

```diff
@@ -0,0 +1,554 @@
+  Your browser does not support the video tag.
+</video>
+
+### Colorize `ls` output
+Colorize the output of `ls` so it's easy to tell the difference between files and directories.
+
+<img src="images/cli026.png" class="invert-colors-in-dark-mode" width="768px" alt="colorized ls example"/>
+
+#### Windows/WSL and Linux (Bash shell)
+{: .primer-spec-toc-ignore }
+
+Verify you're using the Bash shell, typical on WSL Ubuntu Linux.
+```console
+$ echo $0
+-bash
+```
+{: data-variant="no-line-numbers" }
+
+Edit your shell customization file.
+```console
+$ wslview ~/.bash_profile
+```
+{: data-variant="no-line-numbers" }
+
+<div class="primer-spec-callout warning" markdown="1">
+**WSL Pitfall:** You may need to install `wslu`, which includes `wslview`.
+```console
+$ sudo apt install wslu
+```
+</div>
+
+Add this line.  Whenever you type `ls`, you'll actually get `ls --color`, which adds color.
+```bash
+alias ls='ls --color'
+```
+{: data-variant="no-line-numbers" data-title="~/.bash_profile" }
+
+Close your terminal and reopen it.  You should see colorized `ls` output.
+
+<img src="images/cli027.png" class="invert-colors-in-dark-mode" width="512px" alt="colorized ls WSL example"/>
+
+#### macOS (Z shell)
+{: .primer-spec-toc-ignore }
+
+Verify you're using the Z shell, typical on macOS.
+```console
+$ echo $0
+zsh
+```
+{: data-variant="no-line-numbers" }
+
+Edit your shell customization file
+```console
+$ open ~/.zshrc
+```
+{: data-variant="no-line-numbers" }
+
+Add this line.  Whenever you type `ls`, you'll actually get `ls -G`.
+```bash
+alias ls='ls -G'
+```
+{: data-variant="no-line-numbers" data-title="~/.zshrc" }
+
+Close your terminal and reopen it.  You should see colorized `ls` output.
+
+<img src="images/cli028.png" class="invert-colors-in-dark-mode" width="512px" alt="colorized ls macOS example"/>
+
+### Customize prompt
+Customize the terminal prompt to be more helpful and look prettier.
+
+First, complete the [Colorize `ls` output](#colorize-ls-output) section. At this point, you should know whether you are using Bash or Z Shell, and you should have a working `.bash_profile` or `.zshrc` file.
+
+#### Windows/WSL and Linux (Bash shell)
+{: .primer-spec-toc-ignore }
+
+Add a line to your `.bash_profile` file that sets the `PS1` environment variable.
+```bash
+export PS1='\[\e[90m\][\u] \[\e[36m\]\w/ \[\e[31m\]$ \[\e[0m\]'
+```
+{: data-variant="no-line-numbers" data-title="~/.bash_profile" }
+
+Close your terminal and reopen it.  It should look like this.  For more, check out [this guide](https://medium.com/@adamtowers/how-to-customize-your-terminal-and-bash-profile-from-scratch-9ab079256380).
+
+<img src="images/cli029.png" class="invert-colors-in-dark-mode" width="768px" alt="customized bash example"/>
+
+#### macOS (Z shell)
+{: .primer-spec-toc-ignore }
+
+Add a line to your `.zshrc` file that sets the `PS1` environment variable.
+```zsh
+PROMPT='%F{green}[%n] %F{blue}%~%f %F{red}$%f '
```

Comment:
How would you feel about making the `$` bold blue?  It would be less distracting to use two colors instead of 3.  It would also be similar to old-school Ubuntu defaults.